versions.properties

```
@@ -11,7 +11,7 @@
 
 # Project versioning, run the following gradle command to update version numbers automatically:
 # ./gradlew bumpVersion --nextVersion=X.Y.Z
-version.klaviyo.versionCode=18
+version.klaviyo.versionCode=19
```

Is this going to be the last commit before we cut the release?

---

Hm not sure why I had this. I haven't reviewed the 3.0.0 release contents yet, so I'm not certain. We must be getting close though

---

```suggestion
version.klaviyo.versionCode=18
```
Lets bump this in a standalone PR